.github/workflows/docs-checks.yml

```diff
@@ -1,109 +0,0 @@
-
-      - name: Reset caching
-        run: python -c "import time; days = time.time() / 60 / 60 / 24; print(f'TIME_PERIOD=d{int(days / 2) * 2}')" >> $GITHUB_ENV
-
-      # Note: This uses an internal pip API and may not always work
-      # https://github.com/actions/cache/blob/master/examples.md#multiple-oss-in-a-workflow
-      - name: Cache pip
-        uses: actions/cache@v2
-        with:
-          path: ~/.cache/pip
-          key: ${{ runner.os }}-docs-test-pip-td${{ env.TIME_PERIOD }}-${{ hashFiles('requirements/pytorch/*.txt') }}
-          restore-keys: |
-            ${{ runner.os }}-docs-test-pip-td${{ env.TIME_PERIOD }}-
-
-      - name: Install dependencies
-        env:
-          PACKAGE_NAME: pytorch
-          FREEZE_REQUIREMENTS: 1
-        run: |
-          sudo apt-get update
-          sudo apt-get install -y cmake pandoc
-          pip --version
-          pip install -q fire
-          # python -m pip install --upgrade --user pip
-          pip install -e . --quiet -r requirements/pytorch/docs.txt --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
-          pip install -r requirements/pytorch/devel.txt
-          pip list
-        shell: bash
-
-      - name: Test Documentation
-        env:
-          SPHINX_MOCK_REQUIREMENTS: 0
-        working-directory: ./docs
-        run: |
-          # First run the same pipeline as Read-The-Docs
-          make doctest
-          make coverage
-
-  make-docs:
-    runs-on: ubuntu-20.04
-    steps:
-      - uses: actions/checkout@v2
-        with:
-          submodules: true
-          # lfs: true
-      - uses: actions/setup-python@v2
-        with:
-          python-version: 3.9
-
-      # Note: This uses an internal pip API and may not always work
-      # https://github.com/actions/cache/blob/master/examples.md#multiple-oss-in-a-workflow
-      - name: Cache pip
-        uses: actions/cache@v2
-        with:
-          path: ~/.cache/pip
-          key: ${{ runner.os }}-docs-make-pip-${{ hashFiles('requirements/pytorch/base.txt') }}
-          restore-keys: |
-            ${{ runner.os }}-docs-make-pip-
-
-      - name: Install dependencies
-        env:
-          PACKAGE_NAME: pytorch
-          FREEZE_REQUIREMENTS: 1
-        run: |
-          sudo apt-get update
-          sudo apt-get install -y cmake pandoc
-          pip --version
-          pip install -e . --quiet -r requirements/pytorch/docs.txt --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
-          # install Texlive, see https://linuxconfig.org/how-to-install-latex-on-ubuntu-20-04-focal-fossa-linux
-          sudo apt-get update && sudo apt-get install -y texlive-latex-extra dvipng texlive-pictures
-          pip list
-        shell: bash
-
-      - name: Make Documentation
-        working-directory: ./docs
-        run: |
-          # First run the same pipeline as Read-The-Docs
-          make html --debug --jobs $(nproc) SPHINXOPTS="-W --keep-going"
```

Comment:
let's keep it with calling `make docs` from root

Reply:
Why would it be necessary? I don't think we need to build docs in any of GHA workflows if we follow #13572.

Reply:
maybe not build but doctest it :) https://github.com/Lightning-AI/lightning/issues/13572#issuecomment-1185727378